sdk/communication/azure-communication-networktraversal/README.md

```diff
@@ -80,10 +80,32 @@ CommunicationRelayClient communicationRelayClient = new CommunicationRelayClient
 ## Examples
 
 ### Getting a new Relay Configuration
+
+<!-- embedme ./src/samples/java/com/azure/communication/networktraversal/ReadmeSamples.java#L124-L135 -->
+```java
+CommunicationRelayClient communicationRelayClient = createCommunicationNetworkTraversalClient();
+CommunicationRelayConfiguration config = communicationRelayClient.getRelayConfiguration();
+
+System.out.println("Expires on:" + config.getExpiresOn());
+List<CommunicationIceServer> iceServers = config.getIceServers();
+
+for (CommunicationIceServer iceS : iceServers) {
+    System.out.println("URLS: " + iceS.getUrls());
+    System.out.println("Username: " + iceS.getUsername());
+    System.out.println("Credential: " + iceS.getCredential());
+    System.out.println("RouteType: " + iceS.getRouteType());
+}
+```
+
+### Getting a new Relay Configuration providing a user
+
 Use the `createUser` function to create a new user from CommunicationIdentityClient
 Use the `getRelayConfiguration` function to get a Relay Configuration
 
-```java readme-sample-getRelayConfiguration
+<!-- embedme ./src/samples/java/com/azure/communication/networktraversal/ReadmeSamples.java#L97-L113 -->
```

Comment:
We are using a new code snippet tool. Consider using the new tool. See https://github.com/Azure/azure-sdk-for-java/issues/24958 for more details.

Reply:
I'm gonna create a follow up task for this and for the RouteType package. For now I'll leave it like this.